cg/services/orders/order_status_service/dto/order_summary.py

```diff
@@ -4,10 +4,11 @@
 class OrderSummary(BaseModel):
     order_id: int = Field(exclude=True)
     total: int
-    delivered: int | None = None
-    running: int | None = None
     cancelled: int | None = None
+    completed: int | None = None
+    delivered: int | None = None
     failed: int | None = None
-    in_sequencing: int | None = None
     in_lab_preparation: int | None = None
+    in_sequencing: int | None = None
     not_received: int | None = None
+    running: int | None = None
```

Comment:
If you have the energy, we can skip marking these as nullable. See https://github.com/Clinical-Genomics/trailblazer/blob/3295e50ebd202c2e6c202e58faca88dbfe2a85f8/trailblazer/services/analysis_service/utils.py#L16

Reply:
All? Or just the ones from TB? @seallard

Reply:
Made them nullable!

Reply:
Now that you say it, I think all!

Reply:
I did all!